sdk/js/src/service/devices/index.js

```diff
@@ -521,12 +521,8 @@ class Devices {
               application_id: applicationId,
             },
           }
-      try {
```

Comment:
This try/catch and wrapping the error will make us loose the error shape we use in the Console to forward errors to the user.